img/u/ucalgary.ca.svg

```diff
@@ -0,0 +1 @@
.2 0 2.1-1 2.1-2.1 0-.2 0-.4-.1-.6 0 0 .1 0 0-.1l29-48.5 29 48.5v.1c-.1.2-.1.4-.1.6 0 1.2 1 2.1 2.1 2.1 1.2 0 2.1-1 2.1-2.1s-.8-2-1.9-2.1l-.5-.8s3-3.3 4-4.8c3.2-4.6 2.8-15.5-7.8-24m-1.9 40.9c-1.2-3.4-3.5-6.9-7-9.5 2.8-.9 6.4-.9 8.8.7-.8-2.1-3.4-4.9-5.3-6.7-.3.2-.7.5-.9.8-.8.8-1.4.1-.9-.6.2-.2.6-.9.9-1.1-.6-.5-1-.9-1.1-1-.4.2-.8.6-1 .9-.8.8-1.4.1-.9-.6.2-.2.7-.9.9-1.1-.5-.4-1-.8-1.4-1.1-.9 1.2-1.9 2.4-3.5 1.7-1.6-.8 0-1.9 0-3.5 0-6.4-4-8.8-5.1-11.6-1-2.7-2-7.6-2.8-8.9-.8-1.3-4.2-2.8-5.7-1.6-1.5-1.2-4.9.2-5.7 1.6-.8 1.3-1.8 6.2-2.8 8.9-1 2.7-5 5.1-5 11.6 0 1.6 1.6 2.7 0 3.5-1.5.8-2.5-.5-3.5-1.7-.4.3-1 .7-1.4 1.1.3.2.7.9.9 1.1.5.7-.2 1.4-.9.6-.3-.3-.7-.6-1-.9-.1.1-.5.5-1.1 1 .3.2.7.9.9 1.1.5.7-.2 1.4-.9.6-.2-.3-.6-.6-.9-.8-1.9 1.8-4.4 4.6-5.3 6.7 2.5-1.7 5.8-1.6 8.7-.6-3.5 2.6-5.7 6.1-6.9 9.4-5.5-1.5-10.6-3.5-14.8-5.8V30.2c0-22.1 31.6-47.7 39.8-50 8.1 2.3 39.6 27.9 39.6 50V44c-4.2 2.4-9.3 4.3-14.7 5.8" class="st2"/></g><g id="g176" transform="translate(179.794 180.95)"><path id="path178" d="m-87.9 6.2-9.5 2v-.9h-2.8v.9l-5.3-.8h-.2l-4-.6v14l.7.2v2.3l1.4-1c1-.7 1.5-.9 1.8-.9.5 0 .8.4 1.3 1 .1.1.2.3.4.4.7.7 1.4 1.1 2.1 1.1.9 0 1.9-.5 3.1-1.6 1.2 1.1 2.2 1.6 3.1 1.6.8 0 1.5-.3 2.1-1.1.1-.1.2-.3.4-.4.5-.6.9-1 1.3-1 .3 0 .9.2 1.8.9l1.4 1V21l.7-.2V6.2zm-24.8 6.8c-.1-.4-.7-.6-1.4-.6 0-.4-.1-.8-.2-.9-.4-.5-4.8-3.6-5.5-3.8h-.3c-.2 0-.4 0-.7.1-.2-.7-.6-1.1-1.1-1.1-.5 0-.9.4-1.1 1.1-.3-.1-.6-.1-.8-.1h-.3c-.6.2-5 3.3-5.4 3.8-.1.1-.2.5-.2.8-.7 0-1.3.2-1.4.6-.1.4.2.9.7 1.3-.3.3-.5.8-.5 1 0 .6 1.7 5.6 2.1 6.2.1.1.4.3.8.5-.2.6-.1 1.2.2 1.4.4.2.9.1 1.4-.2.2.4.6.7.8.7.4.1 2.2.2 3 .2 1.2 0 3.3 0 3.7-.2.2-.1.5-.3.7-.6.6.4 1.1.5 1.5.3.3-.2.4-.8.2-1.4.4-.1.8-.4.9-.5.4-.5 2.1-5.5 2.1-6.2 0-.2-.1-.6-.4-.9 1-.6 1.3-1.1 1.2-1.5m-21.2-6.2-4 .6h-.2l-5.3.8v-.9h-2.8v.9l-9.5-2v14.5l.7.2v2.3l1.4-1c1-.7 1.5-.9 1.8-.9.5 0 .8.4 1.3 1 .1.1.2.3.4.4.7.7 1.4 1.1 2.1 1.1.9 0 1.9-.5 3.1-1.6 1.2 1.1 2.2 1.6 3.2 1.6.8 0 1.5-.3 2.1-1.1.1-.1.2-.3.4-.4.5-.6.8-1 1.3-1 .3 0 .9.2 1.8.9l1.4 1v-2.3l.7-.2.1-13.9zm12.1 23.1c-29.3 0-39.7-9-39.7-9V-5.4c4 2 8.7 3.7 13.8 5.1-.1.5-.2 1-.2 1.5-.1 1.4.1 2.5.8 3.2.5.5 1.1.8 1.8.8.8 0 2.1-.4 3-2.7l.6-1.5c6.8 1.4 13.7 2.1 19.8 2.1 6.2 0 13.1-.7 19.9-2.1l.6 1.5c.8 2.4 2.2 2.7 3 2.7.7 0 1.3-.3 1.8-.8.7-.7.9-1.8.8-3.2 0-.5-.1-1-.2-1.5 5.1-1.3 9.8-3 13.8-5V21c.1 0-10.3 8.9-39.6 8.9" class="st3"/></g><g id="g180" transform="translate(168.046 138.203)"><path id="path182" d="M-87.3 8.3c1.3-.7 2.7-1.8 2.7-3.2 0-1.5-1.3-2.2-2.6-2.9-.7-.4-1.5-.7-2.4-1.3-1.3-.9-5.5-4-5.1-8.3.1-1.5.7-2.3 2.2-3l-.2 1.9c-.1 1.9 1.6 3.5 4.1 5.3 1.1.8 2.4 1.7 4.1 3 9.2 7.4 10.2 17 7.5 21.1-.4.5-.9 1.3-1.5 2.1-1-8.4-5.2-12.6-8.8-14.7" class="st3"/></g><g id="g184" transform="translate(111.15 153.665)"><path id="path186" d="M-84.5 7.6c-.6-.8-1.1-1.6-1.5-2.1-2.8-4.1-1.7-13.7 7.5-21.1 1.7-1.3 3-2.3 4.1-3 2.5-1.7 4.2-3.3 4.1-5.3l-.2-1.9c1.5.7 2.1 1.5 2.2 3 .4 4.3-3.8 7.5-5.1 8.3-.9.6-1.7 1-2.4 1.3-1.4.7-2.6 1.3-2.6 2.9 0 1.4 1.4 2.5 2.7 3.2-3.7 2.1-8 6.3-8.8 14.7" class="st3"/></g></g></g></g></svg>
```

Comment:
![ucalgary.ca.png](https://user-images.githubusercontent.com/3535780/179313898-4261cfd2-e463-4076-a25a-7a072ca9fa11.png)

Reply:
Changed from svg to png in https://github.com/2factorauth/twofactorauth/pull/6860/commits/b532efef7464940bcba80c5cd77c98bd7885da4e